### ISSUE TITLE
Can you include the crawler?

### ISSUE THREAD

**Issue by posixpascal** [umbrae/reddit-top-2.5-million#4]
I may keep this repository up to date, is it possible to include the crawler you used to crawl all the subs?

**Comment by AdityaAnand1** [comment]
Yes please! This would be very useful. Much better than starting from scratch for similar projects.

**Comment by kanihal** [comment]
+1, Please add the crawler
